packages/grid_client/src/client.ts

```diff
@@ -121,6 +126,7 @@ class GridClient {
       process.on("SIGINT", this.disconnectAndExit);
       process.on("SIGUSR1", this.disconnectAndExit);
       process.on("SIGUSR2", this.disconnectAndExit);
+      //TODO move this line to disconnectAndExit
```

Comment:
why is this a todo and not done yet?

Reply:
Will resolve it later in another PR

Reply:
Open and issue and mention it here too

Reply:
https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1781